boards/arm/cc1352p1_launchxl/board_antenna.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2018-2019, Texas Instruments Incorporated
```

Comment:
Please use your own identifier here (unless you are an employee of TI - just going by your profile).